TrackletAlgorithm/InputRouterTop.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TrackletAlgorithm_InputRouterTop_h
+#define TrackletAlgorithm_InputRouterTop_h
+
+#include "InputRouter.h"
+
+
+// at the moment I'm changing this for each link 
+// is that really the only way to do this? 
+
+// only for PS10G_1_A for now 
+// other cases to be added 
+// when I have final link map
+constexpr unsigned int cNMemories = 16; 
+constexpr unsigned int cNEntriesLUT = kSizePhiCorrTablePS;
+
+void InputRouterTop( const BXType hBx
+	, const ap_uint<6> hInputLinkId // link id 
```

Comment:
`hInputLinkId` and `hLinkNMemories` don't seem to be used anywhere (except for being printed out during C-simulation). Can they be removed?

Reply:
Thanks for spotting it ... legacy from older version of InputRouterTop. Will remove.